### DISCUSSION TITLE
Scenario "The Command Line Murders"

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#20] [Scenarios]
Discussions for the [Command Line Murders scenario](https://github.com/fduran/sadservers/tree/main/scenarios/command-line-murders)

**Comment by Artem-Ushenko** [comment]
I found 3 coincidences in memberships and it is correct solution, but  based on clues it must be 4 coincidences

**Comment by JingYenLoh** [comment]
Since the md5sum of the solution is given, it is rather straightforward to brute force each name's md5sum and arrive at the solution in an unintended manner 🤣. Perhaps a different test might be useful?

**Comment by sbonds** [comment]
The 20 minutes allowed didn't really seem like enough time to do this the "correct" way. Perhaps sending a warning to the user in the terminal that time is ending and allowing a command to reset the clock could be useful? This would prevent idle people wasting time but allow slow investigators to continue working.

**Comment by sbonds** [comment]
OK, I figured out that the limit on retries for unpaid accounts is only if you were unsuccessful, which makes the time limit much more tolerable since I can just try again. :-)

**Reply by fduran** [reply]
@sbonds thanks for the feedback and yes time is a bit tight but you can try as many times as you want. Also there's a timer on the running scenario web page. I think is not hard to add a time warning showing in the command line, and having a way to add more time is something to consider, thanks for the idea!

**Comment by SiegeSailor** [comment]
I was shocked when I saw the number of files. It must be fun reading the story, but I didn't think I could finish it within 20 minutes. So I did `tail +8 people | awk '{print $1}' > /home/admin/agent/people.txt` and then ran:

<img width="634" height="202" alt="image" src="https://github.com/user-attachments/assets/25474d67-505d-47fa-aa11-8fc66674482d" />

I didn't find any name that yielded the given answer. Did I do anything wrong?
